examples/design-system/README.md

```diff
@@ -42,10 +42,10 @@ Using Turborepo simplifies managing your design system monorepo, as you can have
 This Turborepo includes the following packages and applications:
 
 - `apps/docs`: Component documentation site with Storybook
-- `packages/@acme/core`: Core React components
-- `packages/@acme/utils`: Shared React utilities
-- `packages/@acme/tsconfig`: Shared `tsconfig.json`s used throughout the Turborepo
-- `packages/eslint-config-acme`: ESLint preset
+- `packages/ui`: Core React components
+- `packages/utils`: Shared React utilities
+- `packages/tsconfig`: Shared `tsconfig.json`s used throughout the Turborepo
+- `packages/eslint-config-custom`: ESLint preset
```

Comment:
Updating the names just to be consistent with the rest of our examples. Also utils wasn't being used at all so removed it.